cumulus_library/studies/core/manifest.toml

```diff
@@ -2,7 +2,7 @@ study_prefix = "core"
 
 [table_builder_config]
 file_names = [
-    "builder_condition_codableconcept.py",
+    "builder_condition_codeableconcept.py",
```

Comment:
whoops whoops whoops

Reply:
You gotta just hack your dictionary to remove the one and add the other :smile: